### PR TITLE
[type-declarations] Add known return type declarations from new instances

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Ready for review :+1:

File: app/bundles/CoreBundle/Helper/PRedisConnectionHelper.php [hunk 1]
```diff
@@ -81,7 +81,7 @@ public static function createClient(array $endpoints, array $inputOptions): Clie
         $replication = $inputOptions['replication'] ?? null;
 
         if ('sentinel' === $replication) {
-            $inputOptions['aggregate'] = fn () => fn ($sentinels, $options) => new SentinelReplication(
+            $inputOptions['aggregate'] = fn () => fn ($sentinels, $options): \Predis\Connection\Aggregate\SentinelReplication => new SentinelReplication(
```
Comment (escopecz) [inline app/bundles/CoreBundle/Helper/PRedisConnectionHelper.php]:
```suggestion
            $inputOptions['aggregate'] = fn () => fn ($sentinels, $options): SentinelReplication => new SentinelReplication(
```

**Review by escopecz — CHANGES_REQUESTED** [review]
I found one small improvement WDYT?

**Comment by TomasVotruba** [comment]
@escopecz Looks good, I've added the commit :+1:  
Feel free to merge with these details next time. I'm ok with any improvement :clap:

**Comment by TomasVotruba** [comment]
@escopecz Btw, could I ask you for rebase before merge if needed? 
It would require some git back and forth work for me, as there is extra commit now.

**Review by escopecz — APPROVED** [review]
I couldn't spot any issue. Thank you Tomas! 👍

**Comment by TomasVotruba** [comment]
Glad to help :+1:

**Pull request merged**